lib/api_sim/views/index.html.erb

```diff
@@ -10,15 +10,16 @@
   </tr>
   </thead>
   <tbody>
-  <% endpoints.each do |endpoint| %>
+  <% endpoints.each do |endpoint|
+    match = endpoint.match_on_body? ? "/#{endpoint.matcher.source}/" : '' %>
```

Comment:
Would it be possible to throw this into a ViewHelpers method?
ERB syntax can be confusing enough, it seems like adding insult to injury to take advantage of the newline behavior here.